site-cookbooks/postgresql/recipes/default.rb

```diff
@@ -0,0 +1,74 @@
+#
+# Cookbook Name:: postgresql
+# Recipe:: default
+#
+# Copyright 2015, YOUR_COMPANY_NAME
+#
+# All rights reserved - Do Not Redistribute
+#
+include_recipe 'apt'
+# include_recipe 'tar'
+
+# packages = %w{
+#   bison
+#   build-essential
+#   flex
+#   zlib1g-dev
+#   libssl-dev
+#   libreadline-dev
+#   libxml2-dev
+#   libxslt-dev}
+
+# This works, but I'm not sure exactly why... which packages below are needed
+# for postgresql-9-4 to install?
+packages = %w{
+  libpq5
+  pgdg-keyring
+  postgresql-client-common
+  postgresql-9.4}
```

Comment:
Some of the guides recommend `libpq-dev` as well but have no reason why (that I have seen). Ill look into it.